tons of docs (#7)
xf8b/documentation: docs/xf8bot/commands/music/currentlyplaying.md

---
layout: default

> title: CurrentlyPlaying Command

Reviewer: space

Reviewer: ```suggestion
title: Currently Playing Command
```

permalink: /xf8bot/commands/music/currentlyplaying/
---

> # CurrentlyPlaying Command

Reviewer: space

Reviewer: ```suggestion
# Currently Playing Command
```

## Info

> The `currentlyplaying` clears the current queue of songs.

Reviewer: definitely clears the queue of songs yes

Reviewer: ```suggestion
The `currentlyplaying` shows the song currently playing in your VC.
```

> The `currentlyplaying` shows the song currently playing in your VC.

Reviewer: ```suggestion
The `currentlyplaying` command shows the song currently playing in the VC the bot is in.
```
